planship/README.md

```diff
@@ -33,13 +33,13 @@ customer = planship.create_customer({
     "email:": "vader@empire.gov"
 })
 
-# Subscribe the customer with a given ID to a plan with a slug "medium"
+# Subscribe the customer to a plan with the slug "medium"
 subscription = planship.create_subscription(customer.id, "medium")
 
-# Retrieve entitlements for a customer with a give ID
+# Retrieve entitlements for the customer
 entitlements = planship.get_entitlements(customer.id)
 
-# Report usage for a customer with given ID for a given metering ID
+# Report 11 units of usage for the "api-call" metering ID for the customer
 planship.report_usage(customer.id, "api-call", 11)
 ```
 
```

Comment:
Change to absolute path as discussed so it works on PyPI

Reply:
Done.